packages/vitest/src/node/projects/resolveProjects.ts

```diff
@@ -321,6 +322,8 @@ function cloneConfig(project: TestProject, { browser, ...config }: BrowserInstan
       providerOptions: config,
       instances: undefined, // projects cannot spawn more configs
     },
+    // If there is no include pattern in browser.instances[], we should use the include pattern from the parent project
+    include: (overrideConfig.include && overrideConfig.include.length > 0) ? [] : clonedConfig.include,
```

Comment:
Let's also have `exclude` and `includeSource` here, they usually all go together as a single source of test files

Reply:
Make sense! Thank you!  d59b896